plugins/home/src/homePageComponents/StarredEntities/Content.tsx

```diff
@@ -64,7 +63,7 @@ export const Content = () => {
                 aria-label="unstar"
                 onClick={() => toggleStarredEntity(entity)}
               >
-                <StarIcon />
+                <StarIcon style={{ color: '#f3ba37' }} />
```

Comment:
@backstage/reviewers do you think it makes sense to export `YellowStar` from `catalog-react` alternative keep this color in the theme? I see the change of the color is changed in multiple places already and it would be nice to keep it consistent. 

https://github.com/backstage/backstage/blob/b6af1b73fcd5774d5f4b29860c3844b214018df6/plugins/catalog-react/src/components/FavoriteEntity/FavoriteEntity.tsx#L29-L33

https://github.com/backstage/backstage/blob/8f16051d466e21d6d33f2b5b47f32fb8ec911dad/plugins/techdocs/src/home/components/Tables/actions.tsx#L24-L28

https://github.com/backstage/backstage/blob/e1460a97c64a7ed7283f8363e82fca9eb6e0f44b/plugins/catalog/src/components/CatalogTable/CatalogTable.tsx#L56-L60

Reply:
Agreed, _something_ needs to be soon :)

I think I'd prefer if it were just one component, with `color="inherit"` (force it to be text-like) and `color="highlight"` (default golden, but can be changed in the theme) or something like that.

As in, if one needs _specifically_ only an icon (no behavior, just visuals, which happens) that would be one component for example called `FavoriteIcon`, and then there would be one `FavoriteIconButton` with the same prop and reacting to clicks etc.

Does that make sense?

Not necessarily a blocker here - could be done as a separate PR I'd say

Reply:
can you add your suggestions to this issue https://github.com/backstage/backstage/issues/10784 please @benjdlambert @freben